src/School.java

```diff
@@ -36,16 +44,72 @@ private static void showOptionOne() {
         String studentName = inputStudent.nextLine();
 
         System.out.println("Grade to which the student enters " + (students.size() + 1) + ": ");
+        ArrayList<String> grade = new ArrayList<>();
+        grade.add("Sixth");
+        grade.add("Seventh");
+        grade.add("Eighth");
+        grade.add("Ninth");
+        grade.add("Tenth");
+        grade.add("Eleven");
+        System.out.println(grade);
```

Comment:
YOU ARE PRINTING NOTHING HERE, list is nothing for the human eye we have been talking about this many times